connector.go

```diff
@@ -250,8 +251,12 @@ func (wrb *WapiRequestBuilder) BuildUrl(t RequestType, objType string, ref strin
 		qry = vals.Encode()
 	}
 
+	scheme := "https"
```

Comment:
Can we add validation to not support other schemes?

Reply:
making a change to just allow https or http